media-gfx/solvespace/solvespace-2.3.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+DXFRW_COMMIT="87ff1082f49b1286a033ac8f38fe8dde0975bb8b"
+DXFRW_PV="0.6.3"
+DXFRW_PN="libdxfrw"
+DXFRW_P="${DXFRW_PN}-${DXFRW_PV}-${DXFRW_COMMIT}"
+
+inherit cmake-utils
+
+DESCRIPTION="Parametric 2d/3d CAD"
+HOMEPAGE="http://solvespace.com"
+SRC_URI="https://github.com/solvespace/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/solvespace/${DXFRW_PN}/archive/${DXFRW_COMMIT}.zip -> ${DXFRW_P}.zip"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-cpp/gtkmm:2.4
+	dev-cpp/pangomm:1.4
+	dev-libs/json-c:=
+	media-libs/fontconfig
+	media-libs/freetype:2
+	media-libs/glew:0=
+	media-libs/libpng:0=
+	virtual/opengl"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+GIT_COMMIT_HASH="4d1e1341d926ac356b295d5cc3d61c7a6cd7d07d"
```

Comment:
Is this really useful? I'd say it'll be really easy to miss updating it.

Reply:
I think it's indifferent in this case. 

> src_configure() {
> ...
>         -DGIT_COMMIT_HASH=${GIT_COMMIT_HASH}

It's same challenge to attantion like  

> src_configure() {
> ...
>         -DGIT_COMMIT_HASH="4d1e1341d926ac356b295d5cc3d61c7a6cd7d07d"

Reply:
Let me rephrase. Is there any value in passing `-DGIT_COMMIT_HASH` at all? Can you skip it, leave it empty or something like that?

Reply:
Or value pass via -D or patch CMakeLists.txt with set this variable. It cannot be empty or skipped at all.

Reply:
Can you put `${PV}` there instead? That will at least update itself automatically.

Reply:
Ok, so maybe move it to the top and add comment saying you need to update it on bump.